ProjectTraning/Car.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning
+{
+    class Car : Vehicles
```

Comment:
Completely the same comments as for Bicycle belong to this class and should be fixed.